packages/salesforcedx-vscode-core/src/context/workspaceOrgType.ts

```diff
@@ -0,0 +1,59 @@
+import * as path from 'path';
+import * as vscode from 'vscode';
+
+import {
+  ForceConfigGet,
+  ForceOrgList
+} from '@salesforce/salesforcedx-utils-vscode/out/src/cli';
+
+export async function setupWorkspaceOrgType() {
```

Comment:
I wasn't sure where to put these methods, so I put them in a new `context` directory. I'm totally open to suggestions on where they really belong.